scripts/uberenv/packages/axom/package.py

```diff
@@ -0,0 +1,547 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+import os
+import socket
+from os.path import join as pjoin
+
+import llnl.util.tty as tty
+
+
+def cmake_cache_entry(name, value, comment=""):
+    """Generate a string for a cmake cache variable"""
+    return 'set({0} "{1}" CACHE PATH "{2}")\n\n'.format(name, value, comment)
+
+
+def cmake_cache_option(name, boolean_value, comment=""):
+    """Generate a string for a cmake configuration option"""
+
+    value = "ON" if boolean_value else "OFF"
+    return 'set({0} {1} CACHE BOOL "{2}")\n\n'.format(name, value, comment)
+
+
+def get_spec_path(spec, package_name, path_replacements={}, use_bin=False):
+    """Extracts the prefix path for the given spack package
+       path_replacements is a dictionary with string replacements for the path.
+    """
+
+    if not use_bin:
+        path = spec[package_name].prefix
+    else:
+        path = spec[package_name].prefix.bin
+
+    path = os.path.realpath(path)
+
+    for key in path_replacements:
+        path = path.replace(key, path_replacements[key])
+
+    return path
+
+
+class Axom(CMakePackage, CudaPackage):
+    """Axom provides a robust, flexible software infrastructure for the development
+       of multi-physics applications and computational tools."""
+
+    maintainers = ['white238']
+
+    homepage = "https://github.com/LLNL/axom"
+    git      = "https://github.com/LLNL/axom.git"
+
+    version('main', branch='main', submodules=True)
+    version('develop', branch='develop', submodules=True)
+    version('0.3.3', tag='v0.3.3', submodules="True")
```

Comment:
Should 0.4.0 be listed in axom/package.py?

Reply:
I guess so, although, this tag does not exist yet, but, will be created as soon as this branch gets merged -- I just added it.